code/routers/user/profile.js

```diff
@@ -33,14 +36,14 @@ router.get('/profile', (req, res) => {
 
   User.findById(userID, function(err, user) {
 
-    if(err) {
+    if(err || user==null) {
       return res.status(profileCodes.profile.profileNotFound.status).send({
         message: profileCodes.profile.profileNotFound.message
       });
     }
     else{
-      userData.push({firstName: user.firstName, lastName: user.lastName,
-                signupDate: user.signupDate, locationID: user.locationID});
+      userData = {firstName: user.firstName, lastName: user.lastName,
```

Comment:
What's the difference in doing this?

Reply:
This way it is returned as a single JSON object instead of a list with 2 JSON objects. With just one JSON it will be a bit easier to work with.

Reply:
Got it. Thanks for the clarification!